### PR TITLE
Beginner tutorial tweaks

### PR DESCRIPTION
**Pull request by meritmalling**
Some ideas for updates to the beginner tutorial, corresponding getting-started repo updates [here](https://github.com/FlatFilers/getting-started/pull/45).

I could comment on every line of this explaining the driving idea behind it - but they are just those: ideas, lmk what _if anything_ should stick. In lieu of a million comments, driving principles were:

- Use whitespace for linear hierarchy
- Funnel down one path
- Add additional narrative context for the user _and_ to help train our docs
- Include every click, etc as call out step

### CODE REVIEW

File: quickstart/meet-the-workbook.mdx [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
+  To go through this tutorial you'll need a Flatfile account. We'll be using the
+  credentials from your free account to assemble your first data import
+  experience.
+</Note>
 
-## About the Workbook
+In your account use the quickcopy button to copy your secrets key. You're about to use this credential to create a Workbook in your account.
 
-A <Tooltip tip="Analogous to a database...">[Workbook](https://flatfileinc.mintlify.dev/docs/concepts/workbooks)</Tooltip> replaces the spreadsheet template(s) that you may currently share with others when requesting data. Unlike a spreadsheet, a Workbook enables your users to import, validate, and correct data before sending it back to you.
+INSERT SCREENSHOT HERE
```
Comment (brentkulwicki) [inline quickstart/meet-the-workbook.mdx:17]: Do you have a screenshot for this??

File: quickstart/submit-action.mdx [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
 
-When the button is clicked by a user, a job is created. We will [listen for that job to be ready](../concepts/events#job-ready), and our code will call out to an external webhook receiver provided by the [webhook.site](https://webhook.site) service.
+In this step of the tutorial, we're going to configure our Workbook's primary Action, defined in our Blueprint as the `submitAction`.
 
-1. To create the receiver, navigate to [webhook.site](https://webhook.site)
-1. Copy the value of the path after webhook.site for `Your unique URL`.
-1. Next, add the following to your default export function.
+The primary Action button is located in the upper right corner of your Workbook.
+
+INSERT SCREENSHOT HERE
```
Comment (brentkulwicki) [inline quickstart/submit-action.mdx:22]: Same here. Is there a screenshot missing?

**Review by brentkulwicki — APPROVED** [review]
A couple of comments about maybe some missing screenshots, but otherwise, this is 🔥

File: quickstart/meet-the-workbook.mdx (hunk 1 above)
Comment (meritmalling) [reply quickstart/meet-the-workbook.mdx:17]: Oh shoot, I must have lost in rebase, lmk add ty!

File: quickstart/submit-action.mdx (hunk 2 above)
Comment (meritmalling) [reply quickstart/submit-action.mdx:22]: Same! They're all gone! 😱

**Pull request merged**